buzzmobile/simulation/sim_car_interface/src/sim_car_interface.cpp

```diff
@@ -39,7 +39,8 @@ constexpr double chassis_length = 1.8;
 constexpr double chassis_width = 1.57;
 constexpr double inv_chassis_length = 1.0 / chassis_length;
 constexpr double chassis_width_2 = chassis_width / 2.0;
-constexpr double max_torque = 2.5; //lololol
+//TODO(sahit): this can also be calculated/measured when we have the car
+constexpr double max_torque = 2.5; 
```

Comment:
All the measurement ones can be the same issue. Just be sure to link to each line where these TODOs exist